build.gradle

```diff
@@ -46,16 +46,16 @@ dependencies {
     implementation 'org.slf4j:slf4j-api:2.0.5'
     implementation 'com.google.api.grpc:proto-google-common-protos:2.14.3'
     implementation 'com.squareup.okhttp3:okhttp:4.12.0'
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.12.0'
```

Comment:
I should have updated this to 4.12.0 when updating `com.squareup.okhttp3:okhttp` to 4.12.0 in the previous PR